cardinal/bot.py

```diff
@@ -99,9 +99,20 @@ def signedOn(self):
 
         # Set the currently connected network
         self.network = self.factory.network
-
+        self.network_alias = self.factory.network.split('.')[-2]
+
+        if self.network_alias == 'quakenet':
```

Comment:
geographically-specified should not break this as I use the [-2]
using send though sounds reasonable :)